db/migrate/20171208151635_add_metric_to_events.rb

```diff
@@ -0,0 +1,6 @@
+class AddMetricToEvents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :events, :metric, :integer
```

Comment:
game_mode is supposed to handle what you are calling metric right now the tournament saves its tournament type(KO, double-KO) in the column. The League saves the league type(round robin, double round robin, etc.) in that column, so I would suggest that the rankinglist saves what system it uses in that column aswell (Elo, trueskill, or whatever).

Reply:
that's a nice suggestion